Maestro/subscriptions.json

```diff
@@ -630,6 +630,35 @@
           ]
         }
       }
+    },
+    // Update dependencies in WCF release/uwp6.0
+    {
+      "triggerPaths": [
+        "https://github.com/dotnet/versions/blob/master/build-info/dotnet/projectn-tfs/release/uwp6.0/Latest.txt",
+        "https://github.com/dotnet/versions/blob/master/build-info/dotnet/standard/release/uwp6.0/Latest.txt",
```

Comment:
dotnet/standard release/uwp6.0 doesn't exist. Perhaps @dagood could comment on this. We should either remove this or update to listen to release/2.0.0. I don't have the context to recommend one over the other.